core/source.ts

```diff
@@ -83,9 +97,8 @@ export default class SiteSource implements Source {
   loadDirectory(directory: Directory = this.root) {
     const path = this.site.src(directory.src.path);
 
-    return concurrent(
-      Deno.readDir(path),
-      (entry) => this.#loadEntry(directory, entry),
+    return concurrent(Deno.readDir(path), (entry) =>
+      this.#loadEntry(directory, entry),
```

Comment:
This was prettier, nothing to see here really.

I edited this file several times and at some point I triggered it by accident.